### DISCUSSION TITLE
ideas for new update

### DISCUSSION THREAD

**Discussion by ABIGDachshunch** [diamond-rip/Eden#90] [Ideas]
event should be adding some custom gamemode
like :
Deathrun,Stoplight,Knockout,4Corners
Also can add map for even...

**Comment by ABIGDachshunch** [comment]
also survival game event...

**Comment by ABIGDachshunch** [comment]
I can send u minehq SG that my friend used to skid

**Comment by GoodestEnglish** [comment]
LMAO no. Event is going to have more but not now. Feel free to clone the repo and edit the code by yourself if you wish to add extra events.
Eden will not use other project's code without author's permissions.
